client/src/pages/Login.js

```diff
@@ -0,0 +1,136 @@
+import React, { useState } from "react";
+import axios from "axios";
+import { useHistory } from "react-router-dom";
+
+import LoginRegisterNavbar from "./LoginRegisterNavbar";
+
+import {
+  Button,
+  CssBaseline,
+  TextField,
+  Box,
+  Typography,
+  makeStyles,
+  Container
+} from "@material-ui/core";
+
+const useStyles = makeStyles(theme => ({
+  "@global": {
+    body: {
+      backgroundColor: theme.palette.common.white
+    }
+  },
+  paper: {
+    marginTop: theme.spacing(8),
+    display: "flex",
+    flexDirection: "column",
+    alignItems: "center"
+  },
+  avatar: {
+    margin: theme.spacing(1),
+    backgroundColor: theme.palette.secondary.main
+  },
+  form: {
+    width: "100%",
+    marginTop: theme.spacing(1)
+  },
+  submit: {
+    margin: theme.spacing(3, 0, 2),
+    borderRadius: "25px"
+  },
+  formInput: {
+    [`& fieldset`]: {
+      borderRadius: "25px"
+    }
+  },
+  error: {
+    color: "red",
+    display: "flex",
+    justifyContent: "center"
+  }
+}));
+
+const Login = () => {
+  const classes = useStyles();
+  const history = useHistory();
+  const [loginError, setLoginError] = useState("");
+
+  const onSubmitHandler = event => {
+    setLoginError("");
+    event.preventDefault();
+
+    axios
+      .post("http://localhost:4000/login", {
```

Comment:
You should be able to just use "/login" by going into the package.json of the client folder and changing the proxy from http://localhost:3001 to http://localhost:4000

Reply:
Didn't know that! Nice